content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -594,6 +594,13 @@ appropriate arguments.
 More options and information about the
 [`kubeadm reset command`](/docs/reference/setup-tools/kubeadm/kubeadm-reset/).
 
+## Self-hosting the control plane of a cluster
```

Comment:
on line 47 of this doc and a few other places Self-hosting is referred too and i'm not sure if I should update it or not. aka, the above says it is in alpha, I don't think we have plans to do anything more with it at this point. Should it just be removed from the list?

Reply:
the idea i think, is to remove reference for self-hosting from all places. only create a top level doc self-hosting.md under setup/independent/ that contains information about it.

Reply:
i guess this includes removing:
> Self-hosting | alpha

from the table too.

Reply:
Ok cool! :+1: I will update it when I get a chance. Thank you.

Reply:
Personally, I'm not quite sure why we even keep docs for self-hosting at this point.

It was an alpha feature that we no longer support, and afaict there is no one actually supporting the feature.

Reply:
i think the idea was to have a document under setup/independent, have a clear notice on top that this is unsupported and unmaintained.

if / once we remove the alpha command we can also remove this document.
this was my understanding from a kubeadm office hours meeting.

/assign @timothysc